### DISCUSSION TITLE
Frame of reference - moments

### DISCUSSION THREAD

**Discussion by fernandafenelon** [JSBSim-Team/jsbsim#590] [Q&A]
Hello everyone!

in which frame shall I provide the aerodynamic moments (stability, wind or body) PITCH, YAW and ROLL?
I am using LIFT, DRAG and SIDE as forces (wind axis).

**Comment by seanmcleod** [comment]
JSBSim supports all 3, i.e. `stability`, `wind`, `body`, so use whichever one you have the data for.

**Comment by fernandafenelon** [comment]
I think there might be an issue with the moments output. When I sum the moments inputs given in xml, I am supposed to find a value that equals **moments/roll-wind-aero-lbsft**  but I am getting results that equal to **moments/l-aero-lbsft**.

**Comment by seanmcleod** [comment]
Well only if your moments specified in the aircraft xml are specified for the `wind` axes frame.

Take a look at this parallel discussion that has been taking place at virtually the exact same time - https://github.com/JSBSim-Team/jsbsim/discussions/588

**Comment by seanmcleod** [comment]
@fernandafenelon are you and @aregis74 working together on the same project? Just wondering given you were both asking virtually the same question at the same time.

**Reply by fernandafenelon** [reply]
Yes, we are! lol

**Reply by aregis74** [reply]
Yes! lol.
 Thank you for all your help, @seanmcleod!

**Comment by seanmcleod** [comment]
If you coordinate your questions then I won't have to answer them twice 😉

**Reply by fernandafenelon** [reply]
Ok, sorry about that. Thank you.
